server/views/pages/functionalSkills/macros/sortableTable.njk

```diff
@@ -0,0 +1,29 @@
+{#  -------------------------------------------------------------
+    Sortable table for the functional skills page
+    `config` argument is an object with the following properties:
+      * id - the id of the sortable table
+      * assessmentType - the name of the data array to loop through { englishSkills, mathsSkills, digitalSkills }
```

Comment:
I'm not keen on the parameter name `assessmentType` or your description here, because it suggests the arg is a single value which is the assessment type, where the 3 possible values are `englishSkills`, `mathsSkills` or `digitalSkills` as string literals / enums
But that's not what it is at all 😁 

How about calling it `assessments`, with a comment such as:
```
* assessments - an array of assessment data, where each item is an instance of view model type Assessment
```